UnitTesting/UnitTest1.cs

```diff
@@ -9,7 +9,7 @@ namespace UnitTesting
     public class UnitTest1
     {
         [TestMethod]
-        public void getSubreddit()
```

Comment:
Don't believe this should have changed as it's a method name and should be capitalized